src/vstest.console/Internal/MSBuildLogger.cs

```diff
@@ -100,32 +139,24 @@ private void TestResultHandler(object? sender, TestResultEventArgs e)
         switch (e.Result.Outcome)
         {
             case TestOutcome.Passed:
+                SendMessage("test-passed",
+                    CommandLineResources.PassedTestIndicator,
+                    e.Result.TestCase.DisplayName,
+                    e.Result.Duration.TotalMilliseconds.ToString(CultureInfo.InvariantCulture),
+                    FormatOutputs(e.Result));
+                break;
             case TestOutcome.Skipped:
-
-                var test = e.Result.TestCase.DisplayName;
-                var outcome = e.Result.Outcome == TestOutcome.Passed
-                    ? CommandLineResources.PassedTestIndicator
-                    : CommandLineResources.SkippedTestIndicator;
-                var info = $"++++{outcome}++++{ReplacePlusSeparator(test)}";
-
-                Debug.WriteLine(">>>>MESSAGE:" + info);
-                Output.Information(false, info);
+                SendMessage("test-skipped",
+                    CommandLineResources.PassedTestIndicator,
+                    e.Result.TestCase.DisplayName,
+                    e.Result.Duration.TotalMilliseconds.ToString(CultureInfo.InvariantCulture));
                 break;
             case TestOutcome.Failed:
                 var result = e.Result;
                 Debug.WriteLine(">>>>ERR:" + result.ErrorMessage);
                 Debug.WriteLine(">>>>STK:" + result.ErrorStackTrace);
```

Comment:
Shall it be removed? It doesn't look like something that should go in prod.

Reply:
This won't compile into prod, and I want to be using this locally for debugging possible issues we introduce. Imho good to go.